examples/amp-consent/cmp-vendors.amp.html

```diff
@@ -269,6 +269,20 @@ <h3>Image that is NOT blocked by consent</h3>
         </div>
       </amp-consent>
       <!-- End SourcePoint example -->
+
+      <!-- opencmp example -->
+      <amp-consent id='consent' layout='nodisplay' type='opencmp'>
+        <script type="application/json">
+          {
+            "postPromptUI": "opencmp-consent-prompt-ui"
```

Comment:
Just to verify, you do not want to pass a `clientConfig` object to the consent iframe when it is created?

Reply:
At the moment we don't need one. Can we add that later when needed or will that add refactoring effort for publishers already using our CMP?

Reply:
You can add it later and it will require a refactor from the publishers using your CMP. I am ok with either solution as long as you are aware of the trade offs.

Reply:
Ok, I added `clientConfig` as an empty object.